src/views/QuizView/QuizReview.vue

```diff
@@ -27,7 +27,7 @@
 </template>
 
 <script>
-import "core-js/fn/array/flat-map";
+import "core-js/features/array/flat-map";
```

Comment:
The updated path here throws an error for me, does this happen for you as well?

Reply:
try doing a new npm install on this branch if you haven't already